AspNetCore.Demo.Net6/packages.lock.json

```diff
@@ -73,23 +73,23 @@
       },
       "FSharp.Compiler.Service": {
         "type": "Transitive",
-        "resolved": "43.8.300",
-        "contentHash": "CoPjQYXXwmYkkHm+yxBHSW9IJVLpvwkKGEzXa5A6ebf8v6GfYaxZc5G+VHojDr586oezp1elFemu+A1WWH095A==",
+        "resolved": "43.9.101",
+        "contentHash": "Mb7g68UXRLBmeDdrDRpPfjN/HWxTWr9WwqL9lt2PaRPflo8Qg5uxn479Hk4wYCv9dfNEpUy21KZOfCGsNXterw==",
         "dependencies": {
-          "FSharp.Core": "[8.0.300]",
+          "FSharp.Core": "[9.0.101]",

```

Comment:
Do these lock files need to be updated after a4779588471a9bb9447e0c69a41a58fd4d155245, too?